libs/shared/util-auth/src/User.ts

```diff
@@ -2,8 +2,8 @@
  * Contains information that can be used to identify a user,
  * and also the [[roles]] (only the names) the user has.
  */
-export interface User<ID extends number | string> {
-    id: ID;
+export interface User<I> {
```

Comment:
Can I have any type?

Reply:
Yes, I realized that there are more id types than just a `string` + `number` combination. In our case, we have `ObjectId`. Since we're not serializing this it doesn't matter if it is not a primitive.